src/styles/variables.scss

```diff
@@ -0,0 +1,6 @@
+$card-margin-right: 48px;
```

Comment:
Consider using a more descriptive variable name, such as $card-spacing, instead of $card-margin-right.